add titanic_learh.py

himetarou opened the pull request:
ファイル生成はうまくいってるみたいだけど、スコアが0になる。
Survivedのところが0.5になっているものがあるから？

himetarou committed:
add itanic_learh.py

ファイル生成はうまくいってるみたいだけど、スコアが0になる。
Survivedのところが0.5になっているものがあるから？

MK-nn commented:
submission file の format は決まっている。
1.0 じゃなくて 1、0.0 じゃなくて0、0.5は論外です。
```
Submission File Format
You should submit a csv file with exactly 418 entries plus a header row. Your submission will show an error if you have extra columns (beyond PassengerId and Survived) or rows.
The file should have exactly 2 columns:

PassengerId (sorted in any order)
Survived (contains your binary predictions: 1 for survived, 0 for deceased)
PassengerId,Survived
892,0
893,1
894,0
Etc.
You can download an example submission file (gender_submission.csv) on the [Data page](https://www.kaggle.com/c/titanic/data).
```

MK-nn commented:
あと必要なcsvとかも全部このレポジトリに含めといてもらえるとレビューやりやすい

himetarou committed:
modify titanic_learn.py and add submission.csv

・モデルを決定木からランダムフォレストに変更
（元のモデルがどうやら回帰木だったようで、目的変数に量的変数をとっていたため。後、ランダムフォレストの方が精度がいいらしい（過学習にならないため？））

・csvファイルも含めました。

今回はきちんとスコアも出て、0.73444だった。　

himetarou commented:
> submission file の format は決まっている。 1.0 じゃなくて 1、0.0 じゃなくて0、0.5は論外です。
> 
> ```
> Submission File Format
> You should submit a csv file with exactly 418 entries plus a header row. Your submission will show an error if you have extra columns (beyond PassengerId and Survived) or rows.
> The file should have exactly 2 columns:
> 
> PassengerId (sorted in any order)
> Survived (contains your binary predictions: 1 for survived, 0 for deceased)
> PassengerId,Survived
> 892,0
> 893,1
> 894,0
> Etc.
> You can download an example submission file (gender_submission.csv) on the [Data page](https://www.kaggle.com/c/titanic/data).
> ```

ちなみにこのちっさいウィンドウみたいなのはどうやって表示させてる？

MK-nn commented:
レビュー依頼の場合、赤枠の部分や @himetarou のように mention することで github の方に通知がいくよ！
![Cursor_と_add_titanic_learh_py_by_himetarou_·_Pull_Request__1_·_himetarou_Titanic_Lesson_🔊](https://github.com/user-attachments/assets/01c99042-63db-4a27-9322-c38763a8d3e3)

MK-nn commented:
`
↑を3つ打ち込むだけです！

MK-nn commented:
> > submission file の format は決まっている。 1.0 じゃなくて 1、0.0 じゃなくて0、0.5は論外です。
> > ```
> > Submission File Format
> > You should submit a csv file with exactly 418 entries plus a header row. Your submission will show an error if you have extra columns (beyond PassengerId and Survived) or rows.
> > The file should have exactly 2 columns:
> > 
> > PassengerId (sorted in any order)
> > Survived (contains your binary predictions: 1 for survived, 0 for deceased)
> > PassengerId,Survived
> > 892,0
> > 893,1
> > 894,0
> > Etc.
> > You can download an example submission file (gender_submission.csv) on the [Data page](https://www.kaggle.com/c/titanic/data).
> > ```
> 
> ちなみにこのちっさいウィンドウみたいなのはどうやって表示させてる？

`
↑を3つ打ち込むだけです！

himetarou commented:
```
なるほど
